### DISCUSSION TITLE
metro_code

### DISCUSSION THREAD

**Discussion by rickthomasjr** [alexreisner/geocoder#1555] [Q&A] [answered]
Is there a way to retrieve metro_code by IP address?

**Comment by alexreisner** [comment, marked as answer]
Yes, these two lookups return a field called `metro_code`:

* `:maxmind_geoip2`
* `:freegeoip`

So you could do something like this:

    Geocoder.configure(ip_lookup: :freegeoip)
    results = Geocoder.search('...')
    results.first.metro_code
